File: Server Files/UpdateItemRequest.php
<?php 

    $con = mysqli_connect("localhost", "id155858_admin", "yU85M638", "id155858_user");

    $Category = $_POST["Category"];
	$itemID = $_POST["itemID"];
	$Name = $_POST["Name"];
	$Ingredients = $_POST["Ingredients"];
	$Allergy = $_POST["Allergy"];
	$Vegan = $_POST["Vegan"];
	$Vegetarian = $_POST["Vegetarian"];
	$Cal = $_POST["Cal"];
	$Spicy = $_POST["Spicy"];
	$Price = $_POST["Price"];
	$Servings = $_POST["Servings"];

    //Creating sql query 
	 $sql = "UPDATE menu SET Category = '$Category', Name = '$Name', Ingredients = '$Ingredients', Allergy = '$Allergy', Vegan = '$Vegan', Vegetarian = '$Vegetarian', Cal = '$Cal', Spicy = '$Spicy', Price = '$Price', Servings = '$Servings' WHERE itemID = '$itemID'";
	 
	 //Updating database table 
	 if(mysqli_query($con,$sql))
	 {
	 	$response["success"] = true;  
    
    	echo json_encode($response);
	 }
	 else
	 {
	 	$response["success"] = false;  
    
    	echo json_encode($response);
	 }
 
	 //closing connection 
	 mysqli_close($con);
?>

File: RestaurantManagementSystem/app/src/main/java/com/mdm/restaurantmanagementsystem/request/updateItemRequest.java
package com.mdm.restaurantmanagementsystem.request;

import com.android.volley.Response;
import com.android.volley.toolbox.StringRequest;

import java.util.HashMap;
import java.util.Map;

import static com.mdm.restaurantmanagementsystem.Config.UpdateItemRequest_URL;

public class updateItemRequest extends StringRequest
{
    private Map<String, String> params;

    public updateItemRequest(String Category, String itemID, String Name, String Ingredients, String Allergy, String Vegan, String Vegetarian, String Servings, String Cal, String Spicy, String Price, Response.Listener<String> listener)
    {
        super(Method.POST, UpdateItemRequest_URL, listener, null);
        params = new HashMap<>();
        params.put("Category", Category);
        params.put("itemID", itemID);
        params.put("Name", Name);
        params.put("Ingredients", Ingredients);
        params.put("Allergy", Allergy);
        params.put("Vegan", Vegan);
        params.put("Vegetarian", Vegetarian);
        params.put("Servings", Servings);
        params.put("Cal", Cal);
        params.put("Spicy", Spicy);
        params.put("Price", Price);
    }

    @Override
    public Map<String, String> getParams()
    {
        return params;
    }
}